src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs

```diff
@@ -300,7 +300,7 @@ private void UpdateTextBox()
 				return;
 			}
 
-			_textBox.Text = Text;
+			_textBox.Text = Text ?? string.Empty;
```

Comment:
`Text` should *never* be null.

Reply:
@JulesEmond I think accepting `null` is likely to be specific to `x:Bind` in UWP. Even in that case, you'll find that `Text` is an empty string rather than a null.

Do you have a UWP repro that shows `AutoSuggestBox` having a null `Text`? (not that it accepts null from x:Bind, I know it does accept null from x:Bind)

Reply:
There is one in the issue that is linked to the PR : https://github.com/unoplatform/uno/issues/8496

Reply:
@JulesEmond The one in the issue accepts a null from x:Bind, and it gets turned into an empty string. The `Text` property is not null there.

Reply:
The issue was specifically with WASM as explained in the issue.

Reply:
@Youssef1313 this change is a workaround for a more complex behavior related to the way x:Bind behaves with text.